pkg/pipelines/bootstrap.go

```diff
@@ -73,12 +87,35 @@ func Bootstrap(quayUsername, baseRepo, prefix string) error {
 		}
 		fmt.Printf("%s---\n", data)
 	}
-
 	return nil
 }
 
 func pathToDownloadedFile(fname string) (string, error) {
-	return homedir.Expand(path.Join("~/Downloads/", fname))
+	return homedir.Expand(path.Join("~", fname))
+}
+
+//to get the github token file name
+func getTokenFileName(o *BootstrapOptions) (string, error) {
+	if o.TokenFileName == DefaultTokenFileName {
+		return pathToDownloadedFile("/Downloads/token")
+	}
+	if strings.HasPrefix(o.TokenFileName, "~") {
+		return pathToDownloadedFile(string(o.TokenFileName[1:]))
+	}
+	return o.TokenFileName, nil
+}
+
+// to get the quay file name
+func getQuayIOAuthFileName(quayUsername string, o *BootstrapOptions) (string, error) {
+
+	if o.QuayIOAuthFileName == DefaultQuayIOAuthFileName {
+		return pathToDownloadedFile("/Downloads/" + quayUsername + "-auth.json")
+	}
+	if strings.HasPrefix(o.QuayIOAuthFileName, "~") {
```

Comment:
This won't actually work....

`~/Downloads/myfile.txt` for example will get chopped to `/Downloads/myfile.txt`, which is not within the user's home directory.

Why not just use `homedir.Expand(o.QuayIOAuthFileName)` ?

This would have been highlighted by tests for `getQuayIOAuthFileName`.